app/index.js

```diff
@@ -82,6 +82,7 @@ const basicAuth = require('./browser/basicAuth')
 const async = require('async')
 const tabs = require('./browser/tabs')
 const settings = require('../js/constants/settings')
+require('./browser/webtorrent')
```

Comment:
Should this module expose an `init()` function that is called below, just like `./browser/tabs` does?